container/sagemaker-tensorflow-inferentia/build_artifacts/sagemaker/tensorflow-serving.js

```diff
@@ -0,0 +1,232 @@
+var tfs_base_uri = '/tfs/v1/models/'
```

Comment:
Why is this JS?

Reply:
Copied from SM TF container. The container using NGNIX which is configured using NGINX-JS.